<launch>
 <param name = "product" type = "string" value = "LD03" />
 <node name="LD03" pkg="ldlidar" type="ldlidar" output="screen">
 </node>
 
</launch>
